core/computed/metrics/time-to-first-byte.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {makeComputedArtifact} from '../computed-artifact.js';
+import {NavigationMetric} from './navigation-metric.js';
+import {MainResource} from '../main-resource.js';
+import {NetworkAnalysis} from '../network-analysis.js';
+
+class TimeToFirstByte extends NavigationMetric {
```

Comment:
what's the plan with TTFB? An earlier comment mentioned always using observed

Reply:
Sorry, I pushed changes without calling them out.

I was digging in to how we use TTFB elsewhere in Lighthouse and found this comment:
https://github.com/GoogleChrome/lighthouse/blob/main/core/lib/dependency-graph/simulator/network-analyzer.js#L210

When I applied the extra RTTs the simulated TTFB got *significantly* more accurate.